etc/profile-m-z/retroarch.profile

```diff
@@ -0,0 +1,46 @@
+# Persistent local customizations
+include retroarch.local
+# Persistent global definitions
+include globals.local
+
+blacklist /boot
+blacklist /opt
```

Comment:
```suggestion
```

Reply:
`/boot` is a hardcoded blacklist, you don't need to repeat it.

`/opt` might me necessary if someone installs it there ...